### DISCUSSION TITLE
No Profils photo

### DISCUSSION THREAD

**Discussion by francoiscrouzet** [archaiodata/thacer#33] [Q&A]
En regardant le code et les nom de fichiers, j'ai l'impression qu'il n'y a aucune photo dans le dossier profils qui pourrait être affiché.
Je me trompe? Une erreur de copier/coller peut-être?

**Comment by francoiscrouzet** [comment]
@AlZorba

**Reply by francoiscrouzet** [reply]
Ah j'ai pas regardé pourquoi, mais je tombe aujourd'hui sur une fiche qui a une photo de profil : http://127.0.0.1:5173/ceram?ID=8261&ANA=THAnull&INV=69

**Comment by AlZorba** [comment]
Fixed
